### PR TITLE
fix: `ironfish stop` is broken - issue #941

### PR DESCRIPTION
**Pull request by wd021**
## Summary

tldr: calling `yarn start stop` [doesn’t work properly.](https://github.com/iron-fish/ironfish/issues/941) Node shutdown function in `src/node.ts` never finishes because it hangs on `this.rpc.stop()`.

There are 2 ways to stop the node:

1. SIGINT / SIGTERM / SIGUS2 which initiates a `gracefulShutdown()` in `ironfish-cli/src/commands/`. This successfully processes `closeFromSignal()` in `./start.ts`, which handles `this.node?.shutdown()`.

1. `yarn start stop`. This initiates `start()` in `ironfish-cli/src/commands/stop`, which calls the sdkClient `stopNode()` function. This never completes as the shutdown() function in `src/node.ts` hangs on `this.rpc.stop()`

## Solution
Add an argument for the shutdown() function, ignoreRpc = false. when calling stopNode() from the sdkClient, we can pass a false value so it ignores the rpc.stop() call and continues with the shutdown() gracefully. 

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by NullSoldier** [comment]
I think instead of doing this, stopping the RPC should just stop all adapters, which in turn should just end all in flight requests.

**Comment by wd021** [comment]
gotcha, makes sense 👍

**Comment by NullSoldier** [comment]
I just put up a real fix here, but we'll give you some testnet credit for the investigation and motivation to fix the bug.
